Python/Product/PythonTools/PythonTools/Options/PythonAnalysisOptionsControl.cs

```diff
@@ -69,11 +69,11 @@ public partial class PythonAnalysisOptionsControl : UserControl {
 
             if (pyService.AnalysisOptions.LogLevel == PythonLanguageClient.LogLevel.Error) {
                 _logLevelCombo.SelectedIndex = 0;
-            } else if (pyService.AnalysisOptions.LogLevel == PythonLanguageClient.LogLevel.Warning) {
+            } else if (pyService.AnalysisOptions.LogLevel == PylanceLogLevel.Warning) {
```

Comment:
I'm getting 11 build errors in this file, more of these types of "enum moved" fixes are needed

Reply:
Merge issues - resolved.